parser-utils/src/lib.rs

```diff
@@ -17,4 +25,13 @@ macro_rules! assert_parses_to {
         let result = par.parse(&$input[..]);
         assert_eq!(result, Ok(($expected, &[][..])));
     }}
-}
\ No newline at end of file
+}
+
+/// `satisfy_unwrap!` makes it a little easier to implement a `satisfy_map`
+/// body that matches a particular `Value` enum case, otherwise returning `None`.
+#[macro_export]
+macro_rules! satisfy_unwrap {
```

Comment:
It seems that `satisfy_unwrap` is both in `lib.rs` and in `parser_utils.rs`. I assume that's not intentional?